src/components/team-members/TeamMember.tsx

```diff
@@ -25,7 +25,7 @@ function TeamMember({ teamMember }: TeamMemberProps) {
           <p>{`${teamMember.firstName} ${teamMember.lastName}`}</p>
           <p>{teamMember.role}</p>
         </div>
-        <button type="button" onClick={handleClick} />
+        <button type="button" onClick={handleClick} aria-label="toggle-information" />
```

Comment:
I'd just make this an icon instead of a button and put the `onClick` on `TeamMemberInformation`, so the entire `<div>` is clickable. Or you can use a transparent button with an icon in it, plus some extra padding. The current button is a bit too small, making it hard for the user to show/hide the detailed information.

Reply:
Nice! Changed it to the first suggestion, just because a transparent button would make the text below not selectable right?